### ISSUE TITLE
Fury

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#4]
Issues noticed with Fury from in-game testing
UT2004  Draw - NA
UT3     Draw - 1.3
UT3Port Draw - 1.0 / 1.20 on UT3 the character's head is even with the top middle thruster which even with 1.20 isn't quite the case on UT2004 so it might actually need to be more
  -[Collision, despite appearing missing is actually the same as in UT3, you actually can walk through the arms in UT3 to be clear, not an issue if you want the exact same thing as UT3 but could probably be considered a bug on both games so I felt was worth mentioning]
  - [Priority]
  - [Unusable, no way to enter it on UT2004 no matter where I try....Map tested in was ONS-Torlan, tried at both the base spawn and mid map spawn points]
  - [Try 1.20 or larger drawscale]
  - [Secondary]
  - [Tertiary]

**Comment by GreatEmerald** [comment]
Thanks! The formatting is a bit off, though. The square brackets should not enclose the item, but rather precede it, that will give a checkbox before it. The "Priority", "Secondary" and "Tertiary" points should not have the square brackets at all since it's a header.

**Comment by HellDragon-HK** [comment]
Yeah sorry, I tried to do what you said but I missed something obviously.
Btw should currently unavailable items be listed such as not being able to find the DarkWalker or Nemesis as well as the missing weapons still from U2?
Also think I caught another bug with UT3 weapons I'll post for you.  Also while watching this current video I just realized the UT3 Sniper fires the exact same shot as the Stinger, on UT3 it's a thick orange beam so to speak from what I remember, do you want little things like that listed as well? 

    On Monday, November 27, 2017 2:19 AM, Dainius Masiliūnas <notifications@github.com> wrote:
 

 Thanks! The formatting is a bit off, though. The square brackets should not enclose the item, but rather precede it, that will give a checkbox before it. The "Priority", "Secondary" and "Tertiary" points should not have the square brackets at all since it's a header.—

**Comment by GreatEmerald** [comment]
There's an edit button for your own posts on GitHub. It's there for a reason, if something is missing or wrong, use it! If you could edit your issues to get the formatting right, that would save me a bunch of time editing the posts myself. You posting them in the first place already saved me a whole bunch of time!

Yes, do post little things like that as well. There's a tag "trivial" for things like that, and they actually make for nice entry bugs that people like you could try fixing on their own.

**Comment by Unre-Alex** [comment]
I remember using Fury, bots were using it too, it functioned like the Raptor. I think I changed something with its collision tho, let me see.

**Comment by HellDragon-HK** [comment]
I'll give it a shot, thanks Alex.

**Comment by GreatEmerald** [comment]
Hm, editing the animations is probably not the best solution if there's a possible code fix. Else you will get version mismatches. The same effect could potentially be achieved using `PrePivot`?

**Comment by HellDragon-HK** [comment]
I don't know what I'm doing in the code GreatEmerald that's best left for you and Gping

**Comment by HellDragon-HK** [comment]
Added link for the Fury's UT3 drawscale ukx edit to the top of list

**Comment by HellDragon-HK** [comment]
Added a link for a Fury sound package if anybody wants to work on getting those used before I get around to doing what I can with it.
